### ISSUE TITLE
[4.0] Remaining install sample data screen

### ISSUE THREAD

**Issue by infograf768** [joomla/joomla-cms#25813]
### Steps to reproduce the issue
Install a clean build.
Install or not some languages, then skip

### Expected result
If there is some sample data to install, get the details and install.
If none is available, then go back to complete installation.

### Actual result
As there is no more sample data to install we get a screen like below where the ini lang strings do not exist anymore as they have been recently deleted.

<img width="758" alt="Screen Shot 2019-08-11 at 09 42 17" src="https://user-images.githubusercontent.com/869724/62832656-2b28ed80-bc32-11e9-9e51-fb77b9155d86.png">

Code concerned is in
`/installation/tmpl/remove/default.php` 
and
`/installation/src/Form/Field/Installation/SampleField.php`

**Comment by Quy** [comment]
Related #20262

**Comment by brianteeman** [comment]
Removing this step is easy BUT @wilsonge do you still want 3pd to be able to include their own sql or not? If you do then I guess we can have a check for the presence of a custom.sql and only display this step if it exists. Please advise - its an easy fix

**Comment by wilsonge** [comment]
As part of https://github.com/joomla/joomla-cms/pull/25692 we auto-install custom.sql if it exists.

https://github.com/joomla/joomla-cms/blob/4.0-dev/installation/src/Model/DatabaseModel.php#L713-L722

Basically I'm taking the opinion that as most template providers already offer downloads of file or sample data the 90% use case is the users downloaded it to install sample data. If the template dev's want more flexibility than that then they'll have to go down the plugin approach.

TL/DR It's fine to just remove the step.

**Comment by ghost** [comment]
Closed as having Pull Request #25828
